manager/integration/tests/common.py

```diff
@@ -1684,7 +1684,12 @@ def node_default_tags():
 
     tag_mappings = {}
     for tags, node in zip(DEFAULT_TAGS, nodes):
-        assert len(node.disks) == 1
+        if DATA_ENGINE == "v2":
+            # if v2 data engine is enabled, both a file system disk
```

Comment:
Can you elaborate on the purpose? For v2 backing image, right?

Reply:
Yes, I have updated the comment to below, because a v2 volume use block disk but a v2 backing image can not function without file system disk so the two types of disks need to coexist if we want to test v2 volume with v2 backing image.
```
            # if the v2 data engine is enabled, both a file system disk
            # and a block disk will coexist. This is because a v2 backing image
            # requires a file system disk to function.
```